scispacy/hearst_patterns.py

```diff
@@ -0,0 +1,526 @@
+hypernym = {"POS": {"IN": ["NOUN", "PROPN", "PRON"]}}
```

Comment:
I see, I now remember that the reason we didn't implement this with a matcher before was because there was no way to match noun chunks without using the dep matcher. I suppose this is fine though

Reply:
Ya, the approach here is to identify individual nouns, and then expand them in to noun chunks afterward, for exactly this reason.